styles.css

```diff
@@ -1 +1,120 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container {
+    display: grid;
+    color: white;
+    padding: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
```

Comment:
Deze is helemaal goed, 50px auto repeat(5, 50px); mag ook, maar dat weet je ook al. En idd minder moeilijk doen en minder typen enzo is wel chill.